lib/presentation/resources/todo_cards/todo_cards.screen.dart

```diff
@@ -0,0 +1,179 @@
+import 'package:clean_architecture_demo/app/di.dart';
+import 'package:clean_architecture_demo/domain/model/todo_model.dart';
+import 'package:clean_architecture_demo/domain/repository/todo_repository.dart';
+import 'package:clean_architecture_demo/presentation/resources/assets_manager.dart';
+import 'package:clean_architecture_demo/presentation/resources/todo_cards/todo_cards_state.dart';
+import 'package:clean_architecture_demo/presentation/resources/todo_cards/todo_crads_cubit.dart';
+import 'package:clean_architecture_demo/presentation/resources/values_manager.dart';
+import 'package:clean_architecture_demo/presentation/utils/ui_utils.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class TodoCardsSecreen extends StatefulWidget {
```

Comment:
@dilumdesilva typo in the name of class.

Reply:
Good catch. Fixed!